backend/src/main/java/com/rikishi/rikishi/model/User.java

```diff
@@ -0,0 +1,12 @@
+package com.rikishi.rikishi.model;
+
+public record User(
+    long id,
+    String name,
+    String lastname,
+    int age,
+    double weight,
```

Comment:
AFAIK the csv was supposed to supply weight categories rather than weights -- those were to be manually added by the app user at a later stage, then compared to the category

Reply:
Update: as discussed, the CSV is to contain the weight of the contestant as well as the weight category which was assigned to them